### PR TITLE
Remove xbrlxe:unexpectedContextContent

### PR DESCRIPTION
**Pull request by austinmatherne-wk**
#### Reason for change
The OIM WG agreed to remove xbrlxe:unexpectedContextContent from xBRL-xml which was not testable due to OIM issue [#419](https://gitlab.xbrl.org/oim/oim/-/issues/419)

#### Description of change
Update impl to align with [updated test suite](https://gitlab.xbrl.org/oim/oim/-/merge_requests/440/diffs) (expected failure IDs are needed until the new suite is published)

#### Steps to Test
* CI

**review**:
@Arelle/arelle

### CODE REVIEW

**Comment by aviary2-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: tests/integration_tests/validation/conformance_suite_configurations/xbrl_oim_1_0.py [hunk 1]
```diff
@@ -7,6 +7,10 @@
         '--httpsRedirectCache',
         '--plugins', 'loadFromOIM',
     ],
+    expected_failure_ids=frozenset([
+        '600-xml/index-xbrl-xml.xml/V-05',
+        '600-xml/index-xbrl-xml.xml/V-06',
```
Comment (brettkail-wk) [inline tests/integration_tests/validation/conformance_suite_configurations/xbrl_oim_1_0.py:12]: Perhaps open a ticket to remind to clean this up after a new OIM zip is published, and perhaps a comment with a link to that issue?

File: arelle/plugin/loadFromOIM.py [hunk 2]
```diff
@@ -3025,16 +3014,10 @@ def validateFinally(val, *args, **kwargs):
...
-            modelXbrl.error("xbrlxe:unexpectedContextContent",
-                            _("Contexts not used for dimensions must not contain content in %(containers)s: %(contexts)s"),
-                            modelObject=contextsWithNonDimContainer,
-                            containers=" or ".join(sorted(containersNotUsedForDimensions)),
-                            contexts=", ".join(sorted(c.id for c in contextsWithNonDimContainer)))
         if contextsWithComplexTypedDimensions:
             modelXbrl.error("xbrlxe:unsupportedComplexTypedDimension",  # this pertains only to xBRL-XML validation (JSON and CSV were checked during loading when loadedFromOIM is True)
                             _("Instance has contexts with complex typed dimensions: %(contexts)s"),
-                            modelObject=contextsWithNonDimContainer,
+                            modelObject=contextsWithComplexTypedDimensions,
```
Comment (brettkail-wk) [inline arelle/plugin/loadFromOIM.py:3020]: To be sure, this is a tangential bug fix, right?

Comment (austinmatherne-wk) [reply arelle/plugin/loadFromOIM.py:3020]: Correct

File: tests/integration_tests/validation/conformance_suite_configurations/xbrl_oim_1_0.py (hunk 1 above)
Comment (austinmatherne-wk) [reply tests/integration_tests/validation/conformance_suite_configurations/xbrl_oim_1_0.py:12]: XT-3052

**Review by derekgengenbacher-wf — APPROVED** [review]

**Comment by derekgengenbacher-wf** [comment]
QA +10 Testing locally succeeded.

**Pull request merged**